server/app.js

```diff
@@ -43,7 +43,7 @@ app.use((req, res, next) => {
 
 app.use("/auth", authRouter);
 app.use("/users", userRouter);
-
+app.use("/profile",profileRouter);
```

Comment:
```suggestion
app.use("/profiles",profileRouter);
```

An article on REST naming convention best practices: https://gearheart.io/articles/restful-api-design-best-practices/

Reply:
Done the changes and thanks for sharing the document its really useful.